.circleci/config.yml

```diff
@@ -93,6 +93,18 @@ jobs:
           docker login -u $DOCKER_USER -p $DOCKER_PASS
           docker push citizenlabdotco/back-essential:$CIRCLE_SHA1
 
+  back-lint:
+    resource_class: small
+    executor:
+      name: cl2-back
+      image-tag: $CIRCLE_SHA1
+    working_directory: /cl2_back
+    parallelism: 4
```

Comment:
We should take the `parallelism` line out. The actual rubocop runtime is low compared to the startup time of the containers and CircleCI, so I think it doesn't pay off to run in parallel. We pay for CI time by the minute, so this also increases costs by (almost) 4.

If I'm wrong and it does pay off, then we should adapt the `run` command for it to divide the files between all running containers, as they way it's configured now they're all running exactly the same thing, x4. See [CircleCI docs](https://circleci.com/docs/2.0/parallelism-faster-jobs/#using-the-circleci-cli-to-split-tests). But I simply wouldn't in this case.

Reply:
Done. When you approve this PR, we can merge and run RuboCop on the master for the first time.